[UNIT] llama-index framework - release 4/3/25 (#22)
huggingface/agents-course: units/en/unit2/llama-index/quiz1.mdx

# Small Quiz (ungraded) [[quiz1]]

> Up to this point, you have understood the key components and tools used in LlamaIndex.

Reviewer: ```suggestion
So far we've discussed the key components and tools used in LlamaIndex.
```
